### PR TITLE
2020 markup japanese translation

### PR DESCRIPTION
**Pull request by ksakae1216**
Please review for me.
Part of #586

### CODE REVIEW

File: src/content/ja/2020/markup.md [hunk 1]
```diff
@@ -0,0 +1,1342 @@
...
+      <tr>
+        <td><code>basefont</code></td>
+        <td class="numeric">0.0092%</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="サンプルのページから特定の要素が見つかる確率が低いこと。", sheets_gid="184700688", sql_file="pages_element_count_by_device_and_element_type_present.sql") }}</figcaption>
+</figure>
+
+私たちは、どのような要素が廃れてしまったのかを知るために、これらの要素を含めています。しかし、`dir`と`basefont`はXHTML1.0(2000)で最後に指定され、もはやHTMLの一部ではありませんが、`rp`の稀な使用（これは<a hreflang="en" href="https://www.w3.org/TR/1998/WD-ruby-19981221/#a2-4">1998年の早い段階で</a>、<a hreflang="en" href="https://html. spec.whatwg.org/multipage/text-level-semantics.html#the-rp-element">still part of HTML</a>に言及されており）は、<a hreflang="en" href="https://www.w3.org/TR/ruby/">Rubyマークアップ</a>があまり普及していないことを示唆しているのかもしれない。
```
Comment (bazzadp) [inline src/content/ja/2020/markup.md:453]: Extra space:

```suggestion
私たちは、どのような要素が廃れてしまったのかを知るために、これらの要素を含めています。しかし、`dir`と`basefont`はXHTML1.0(2000)で最後に指定され、もはやHTMLの一部ではありませんが、`rp`の稀な使用（これは<a hreflang="en" href="https://www.w3.org/TR/1998/WD-ruby-19981221/#a2-4">1998年の早い段階で</a>、<a hreflang="en" href="https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-rp-element">still part of HTML</a>に言及されており）は、<a hreflang="en" href="https://www.w3.org/TR/ruby/">Rubyマークアップ</a>があまり普及していないことを示唆しているのかもしれない。
```

File: src/content/ja/2020/markup.md [hunk 2]
```diff
@@ -0,0 +1,1342 @@
...
+
+## その他
+
+これまで、HTMLの一般的な使い方や、上位の要素や属性の採用について説明してきました。このセクションでは、ビューポート、ファビコン、ボタン、入力、リンクなどの特殊なケースを確認しています。ここで注意しなければならないのは、あまりにも多くのリンクがいまだに「http」のURLを指していることです。
+
+### `viewport`の仕様
+
+[viewport](https://developer.mozilla.org/ja/docs/Mozilla/Mobile/Viewport_meta_tag)メタ要素は、モバイルブラウザでのレイアウトを制御するために使用されます。数年前まではWebページを構築する際に「viewport meta要素を忘れないように」というモットーのようなものがありましたが、やがてこれが一般的になり、「ズームやスケーリングが無効になっていないか確認する」というモットーに変わりました。
+
+ユーザーは、テキストを<a hreflanglang="en" href="https://dequeuniversity.com/rules/axe/4.0/meta-viewport-large">最大500%</a>まで拡大・縮小できる必要があります。これが、<a hreflang="en" href="https://developers.google.com/web/tools/lighthouse">Lighthouse</a>や<a hreflang="en" href="https://www.deque.com/axe/">axe</a>のような人気のあるツールの監査が、`meta name="viewport"`要素内で`user-scalable="no"`が使用され、`maximum-scale`属性値が`5`未満の場合に失敗する理由です。
```
Comment (bazzadp) [inline src/content/ja/2020/markup.md:972]:
```suggestion
ユーザーは、テキストを<a hreflang="en" href="https://dequeuniversity.com/rules/axe/4.0/meta-viewport-large">最大500%</a>まで拡大・縮小できる必要があります。これが、<a hreflang="en" href="https://developers.google.com/web/tools/lighthouse">Lighthouse</a>や<a hreflang="en" href="https://www.deque.com/axe/">axe</a>のような人気のあるツールの監査が、`meta name="viewport"`要素内で`user-scalable="no"`が使用され、`maximum-scale`属性値が`5`未満の場合に失敗する理由です。
```

File: src/content/ja/2020/markup.md [hunk 3]
```diff
@@ -0,0 +1,1342 @@
...
+  sql_file="pages_almanac_by_device_and_html_lang.sql"
+  )
+}}
+
+### コメント
+
+コードにコメントを付けることは一般的に良い習慣であり、HTMLコメントはユーザーエージェントによってレンダリングされることなく、HTML文書にメモを付けるためのものです。
+
+```html
+<!-- This is a comment in HTML -->
```
Comment (bazzadp) [inline src/content/ja/2020/markup.md:143]: As this is a comment and not actual code should we translate it?

File: src/content/ja/2020/markup.md [hunk 4]
```diff
@@ -0,0 +1,1342 @@
...
+
+多くのページでは制作のためにコメントが削除されているでしょうが90パーセンタイルのホームページでは、モバイルでは約73個、デスクトップでは約79個のコメントが使用されており、10パーセンタイルでは約2個のコメントが使用されていることがわかりました。 中央値のページでは、16個（モバイル）または17個（デスクトップ）のコメントが使用されています。
+
+約89％のページには少なくとも1つのHTMLコメントが含まれており、約46％のページには条件付きのコメントが含まれています。
+
+#### 条件付きコメント
+
+```html
+<!--[if IE 8]>
+  <p>This renders in Internet Explorer 8 only.</p>
```
Comment (bazzadp) [inline src/content/ja/2020/markup.md:154]: As this is a comment and not actual code should we translate it?

File: src/content/ja/2020/markup.md [hunk 5]
```diff
@@ -0,0 +1,1342 @@
...
+もうひとつ気になっていたのは、とくに2020年に<a hreflang="en" href="https://caniuse.com/details">`details`と`summary`要素</a>が導入されてから、その使用状況です。これらは使われていますか？　それらは著者にとって魅力的で、人気もあるのでしょうか？　結果的には、テストした全ページのうち0.39%しか使用されていませんでした。ただし必要な場面で、正しい方法で使用されているかどうかを判断するのは難しいので、「人気がある」という表現は間違っています。
+
+ここでは、`details`要素の中で`summary`を使用する簡単な例を紹介します。
+
+```html
+<details>
+  <summary>Status: Operational</summary>
+  <p>Velocity: 12m/s</p>
+  <p>Direction: North</p>
+</details>
```
Comment (bazzadp) [inline src/content/ja/2020/markup.md:331]: Should we translate the text in here?

**Review by bazzadp — CHANGES_REQUESTED** [review]
Thansk @ksakae1216 ! Spotted a few things for you to look over.

File: src/content/ja/2020/markup.md [hunk 6]
```diff
@@ -0,0 +1,1342 @@
...
+iandevlin_bio: Ian Devlinは、優れたセマンティックHTMLやアクセシビリティを提唱するウェブ開発者です。<a hreflang="en" href="https://www.peachpit.com/store/html5-multimedia-develop-and-design-9780321793935">HTML5 マルチメディア</a>についての本を書いたこともありますし、<a hreflang="en" href="https://iandevlin.com/">自身のウェブサイト</a>でウェブやその他のことについて散発的に書いています。現在は、ドイツの<a hreflang="de" href="https://www.real-digital.de/">real.digital</a>でシニアフロントエンドエンジニアとして働いています。
+reviewers: [zcorpan, matuzo, bkardell]
+analysts: [Tiggerito]
+editors: [rviscomi]
+translators: [ksakae]
+discuss: 2039
+results: https://docs.google.com/spreadsheets/d/1Ta7amoUeaL4pILhWzH-SCzMX9PsZeb1x_mwrX2C4eY8/
+featured_quote: 私たちは、生きたHTMLをほぼ完全に採用しようとしており、流行のページはすぐに削除し、フレームワークの採用や廃止も迅速に行っています。しかし、HTMLが与えてくれる選択肢を使い切った形跡はありません。
+featured_stat_1: 85.73%
+featured_stat_label_1: "living" HTML doctypeを使用しているページの割合
```
Comment (bazzadp) [inline src/content/ja/2020/markup.md:17]: What's your thoughts on whether we should translate "living" or not? Here and else where in this chapter when we talk about living HTML spec?

File: src/content/ja/2020/markup.md (hunk 1 above)
Comment (ksakae1216) [reply src/content/ja/2020/markup.md:453]: Thanks, I fixed.

File: src/content/ja/2020/markup.md (hunk 2 above)
Comment (ksakae1216) [reply src/content/ja/2020/markup.md:972]: Thanks, I fixed.

File: src/content/ja/2020/markup.md (hunk 3 above)
Comment (ksakae1216) [reply src/content/ja/2020/markup.md:143]: I agree.
I fixed.

File: src/content/ja/2020/markup.md (hunk 4 above)
Comment (ksakae1216) [reply src/content/ja/2020/markup.md:154]: I agree.
I fixed.

File: src/content/ja/2020/markup.md (hunk 5 above)
Comment (ksakae1216) [reply src/content/ja/2020/markup.md:331]: Yes, I fixed.

File: src/content/ja/2020/markup.md (hunk 6 above)
Comment (ksakae1216) [reply src/content/ja/2020/markup.md:17]: Oh...
Sorry, I translate.

**Review by bazzadp — APPROVED** [review]

**Pull request merged**